src/marqo/s2_inference/s2_inference.py

```diff
@@ -77,6 +77,9 @@ def vectorise(model_name: str, content: Union[str, List[str]], model_properties:
                 raise RuntimeError(f"Vectorise created an empty list of batches! Content: {content}")
             else:
                 vectorised = np.concatenate(vector_batches, axis=0)
+    except IllegalVectoriseError as e:
```

Comment:
The `IllegalVectoriseError` will be raised when the `encode` function is called of `no_model`. This will happen when a user calls add docs or search with content, thus raising a 400.

Reply:
Got it. Thanks!